utils/containers.py

```diff
@@ -63,7 +63,9 @@ def synchronize(self, current_column, column_rows):
         for column in self.columns:
             if column == current_column:
                 continue
-            setattr(column, 'column_rows', column_rows)
+
```

Comment:
It seems like there is already a direct reference to Row objects on each Columns which makes it completely useless to consistently update this list in the Synchronizer. Apparently the data gets updated continuously. Keep checking this.